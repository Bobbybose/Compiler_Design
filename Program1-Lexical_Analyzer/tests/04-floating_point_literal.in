1.2
3.
.4

1e2
1.1e2
1.e2
.1e2

1e+4
3e-5

4.1e+3
4.2e-3
4.e+5
.4e-5