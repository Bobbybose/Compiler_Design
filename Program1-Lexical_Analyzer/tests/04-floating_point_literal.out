floating-point-literal: 1.2: 1, 1
floating-point-literal: 3.: 2, 1
floating-point-literal: .4: 3, 1
floating-point-literal: 1e2: 5, 1
floating-point-literal: 1.1e2: 6, 1
floating-point-literal: 1.e2: 7, 1
floating-point-literal: .1e2: 8, 1
floating-point-literal: 1e+4: 10, 1
floating-point-literal: 3e-5: 11, 1
floating-point-literal: 4.1e+3: 13, 1
floating-point-literal: 4.2e-3: 14, 1
floating-point-literal: 4.e+5: 15, 1
floating-point-literal: .4e-5: 16, 1